### ISSUE TITLE
Accessibility - Individual Country Screen - Talkback Not Reading Entire List of Goods per Country

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#33]
When TalkBack read the list if exploited labor goods it stopped before reading the entire list. I was on the Brazil screen that has 16 exploited goods, and it read the first nine and not the rest of the list. If I manually swipe my way through the list, TalkBack will read all the goods.

**Comment by aquilent-dgemoets** [comment]
This is currently how TalkBack reads scrollable list items (also applies to #50).

**Comment by jfc3-dol** [comment]
Not sure how a person using TalkBack would know that they missed other items in the list unless one of the chimes or dings was that notification similar to how when reading links there is a bell chime at the end of the link.

**Comment by aquilent-dgemoets** [comment]
Yes, I think there are some nuances to using TalkBack that a sighted person would have a hard time realizing/appreciating.  Also, TalkBack is still being improved (I believe), so in time some of the issues we've seen may be addressed.

**Comment by jfc3-dol** [comment]
TalkBack is still not reading goods that are not visually on the screen. Could be an issue with TalkBack so will leave issue open for now.

**Comment by jfc3-dol** [comment]
Again tested and TalkBack is still not reading goods that are not visually on the screen.

Will attempt to look at this with our newer device tomorrow (Wednesday) to see if it's a problem there too.

**Comment by jfc3-dol** [comment]
Even on our new Samsung Galaxy S7 with newer software TalkBack (screen reader) is not reading the items that are off the screen or even mentioning that there are more items.

**Comment by carolynhuang** [comment]
Cannot be resolved. This appears to be a limitation with the technology and having a scrolling feature - TalkBack will only read what is on the screen. The scrolling feature was used so that each country would fall on a separate line. Do you have a solution for this? Otherwise, this cannot be resolved without changing the scrolling feature.

**Comment by jfc3-dol** [comment]
Think we need to rethink a better way to display the longer list of items for the next version of the applications so all users can see all the items. Don't think TalkBack announces the list can be scrolled. If TalkBack did, then the user could possibly scroll the list themselves to see the rest of the list as a stop-gap fix.

**Comment by jfc3-dol** [comment]
Looked at this issue with the latest release of the Android Sweat and Toil and this problem is still an issue.

**Comment by jfc3-dol** [comment]
I am closing this issue because TalkBack now reads all the items in the list of goods on the individual country screen that have more than four or five products.
